security-oauth2/src/main/java/io/micronaut/security/oauth2/configuration/OauthClientConfiguration.java

```diff
@@ -136,4 +136,20 @@ default SecureEndpoint getTokenEndpoint() throws ConfigurationException {
     default AuthorizationServer getAuthorizationServer() {
         return null;
     }
+
+    /**
+     *
+     * @return Whether a request to /.well-known/oauth-authorization-server should be proxied to the authorization server.
+     */
+    default boolean isProxyWellKnownOauthAuthorizationServer() {
```

Comment:
add `@since` to new public methods

Reply:
https://github.com/micronaut-projects/micronaut-security/pull/2024/commits/132e80e160705c04242483beadfd8fd04802e1a0